src/test/java/core/basesyntax/service/fileservice/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service.fileservice;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static FileReaderService fileReader;
+    private static String validFilePath;
+    private static String invalidFilePath;
+    private static List<String> expectedList;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileReader = new FileReaderServiceImpl();
+        expectedList = new ArrayList<>();
+        validFilePath = "src/test/resources/testInput.csv";
+        invalidFilePath = "notExistingPath";
+        expectedList.add("type,fruit,quantity");
+        expectedList.add("b,banana,20");
+        expectedList.add("b,apple,100");
+        expectedList.add("s,banana,100");
+        expectedList.add("p,banana,13");
```

Comment:
looks like this should be inside `readFile_isOk`
so `expectedList` will be a local variable